frontend/src/components/assessment-creation/StatusTag.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Tag } from "@chakra-ui/react";
+
+type StatusProperty = "Draft" | "Published" | "Archived" | "Deleted";
```

Comment:
Can we move this into `types/AssessmentTypes` and then refactor the `status` property of the `AssessmentType` to use this type?